### PR TITLE
docs: README fix links + minor updates

### PR DESCRIPTION
**Pull request by berezovskyi**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -25,22 +25,22 @@ Alternatively, do a manual install:
...
 
-http://localhost:8085/sysml_oslc_server/services/projects/23561420-ef88-4249-bf99-651670ff438f/service6/features/query
+http://localhost:8085/sysml_oslc_server/oslc/projects/23561420-ef88-4249-bf99-651670ff438f/service6/features/query
 
 - `oslc.where sysml:identifier="5bc41f66-4d42-41da-b7a3-92af54dab320"`
 - `oslc.prefix sysml=<http://omg.org/ns/sysml#>`
 
 ## Info
 
-The domain-classes are based on the ecore file, located underhttps
```
Comment (jadelkhoury) [inline readme.md:43]: underhttps???

**Review by jadelkhoury — CHANGES_REQUESTED** [review]
a very small fix.

File: readme.md (hunk 1 above)
Comment (berezovskyi) [reply readme.md:43]: That's your text that I removed :)

Comment (Jad-el-khoury) [reply readme.md:43]: :-)

**Review by Jad-el-khoury — APPROVED** [review]

**Review by jadelkhoury — APPROVED** [review]

**Pull request merged**